### PR TITLE
[FIX] l10n_ar_stock: Se pisan los datos del contacto con los de la orden

### PR DESCRIPTION
**Pull request by mem-adhoc**
Ticket: 74843

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/argentina-sale/pull/182).

File: l10n_ar_stock/views/report_deliveryslip.xml [hunk 1]
```diff
@@ -125,7 +125,7 @@
                     <strong>Deliver to: </strong>
                     <span t-field="partner.name"/>
                 </span>
-                <span t-out="partner" t-options='{"widget": "contact", "fields": ["address", "phone", "mobile"], "no_marker": False, "no_tag_br": True}'/>
+                <span t-out="partner" t-options='{"widget": "contact", "fields": ["address", "phone", "mobile"], "no_marker": False, "no_tag_br": False}'/>
```
Comment (zaoral) [inline l10n_ar_stock/views/report_deliveryslip.xml]: update manifest

**Review by zaoral — CHANGES_REQUESTED** [review]

File: l10n_ar_stock/views/report_deliveryslip.xml (hunk 1 above)
Comment (zaoral) [reply l10n_ar_stock/views/report_deliveryslip.xml]: test do not pass no_tag_br at all

**Comment by zaoral** [comment]
@roboadhoc r+
@roboadhoc override=ci/runbot-oba

we need to override because external runbot error is not linking to the build

**Pull request closed without merging**